indico/modules/events/papers/controllers/management.py

```diff
@@ -111,3 +114,28 @@ def _process(self):
             return jsonify_data()
         return WPManagePapers.render_template('management/competences.html', self._conf, event=self.event_new,
                                               form=form)
+
+
+class RHManageReviewingSettings(RHManagePapersBase):
+    def _process(self):
+        defaults = FormDefaults(content_review_questions=self.event_new.cfp.content_review_questions,
+                                layout_review_questions=self.event_new.cfp.layout_review_questions,
+                                **paper_reviewing_settings.get_all(self.event_new))
+        form = PaperReviewingSettingsForm(event=self.event_new, obj=defaults)
+        if form.validate_on_submit():
+            data = form.data
+            content_review_questions = data.pop('content_review_questions', [])
+            layout_review_questions = data.pop('layout_review_questions', [])
+            self.event_new.paper_review_questions = []
```

Comment:
Right now, saving this form will delete all questions for a review type if it has been disabled.